packages/models-library/src/models_library/settings/docker_registry.py

```diff
@@ -0,0 +1,26 @@
+from typing import Optional
+
+from pydantic import Field, SecretStr
+
+from .base import BaseCustomSettings
+
```

Comment:
now we have settings-library for these things. you did the PR review a few days ago
Actually ``models_library.base.BaseCustomSettings``

Reply:
@pcrespov could you link me to that file. And now I am super confused about this one.
There seem sto be package `settings-library` and there is the submodule `models-library.settings` inside `models-library` which are similar. What happened here?

Reply:
Never mind. I've figured it out. It was placed in the wrong module.